General.ruleset

```diff
@@ -0,0 +1,119 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Isn't https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1401.md the same as CA1051 that you disabled?

Reply:
Disable SA1401 also then.